app/tests/src/tokenRegistry.js

```diff
@@ -111,6 +111,7 @@ describe('Token registry api', () => {
     const expected = new Map(
       Object.entries({
         '1d7f33bd23d85e1a25d87d86fac4f199c3197a2f7afeb662a0f34e1e776f726c646d6f62696c65746f6b656e': {
+          name: 'World Mobile Token',
```

Comment:
not sure why the test was working fine before, seems like the upgrade of webpack led to `Map`s being compared properly